packages/utilities/src/Async.ts

```diff
@@ -49,7 +49,7 @@ export class Async {
     if (this._immediateIds) {
       for (id in this._immediateIds) {
         if (this._immediateIds.hasOwnProperty(id)) {
-          this.clearImmediate(id);
+          this.clearImmediate(parseInt(id, 10));
```

Comment:
Seems like there is a bug here, but I don't have enough context. Id is a string since it's a key in _timeoutIds, but it's being used as a number...